### DISCUSSION TITLE
Scenario "Buenos Aires": Kubernetes Pod Crashing

### DISCUSSION THREAD

**Discussion by fduran** [SadServers/sadservers#36] [Scenarios]
Discussions about scenario [Buenos Aires](https://github.com/fduran/sadservers/tree/main/scenarios/buenos-aires)

**Comment by imranismail** [comment]
The problem with this solution is that describing the pod doesn't give sufficient information to troubleshoot the issue. We also need the ability to see the pod logs to determine the error.

Suggest to add `get/watch pod/logs` permission to the user.

**Comment by Ismail-Shaker** [comment]
<details><summary>Spoiler</summary>
<p>
We can just give the file a 777 permission set and the scenario test check passes successfully, needs to be taken into consideration IMO

</p>
</details>
